packages/strapi-admin/admin/src/components/DownloadInfo/styles.scss

```diff
@@ -9,7 +9,7 @@
   padding-top: 3rem;
   // color: #F64D0A;
   text-align: center;
-  font-family: 'Lato', sans-serif;
+  font-family: Lato;
```

Comment:
> You should always include at least one generic family name in a font-family list, since there's no guarantee that any given font is available. This lets the browser select an acceptable fallback font when necessary.

https://developer.mozilla.org/en-US/docs/Web/CSS/font-family